lib/index.ts

```diff
@@ -680,6 +760,19 @@ export class Worker {
 					typeContract.data.schema,
 					objectWithLinks as any,
 				);
+
+				if (
+					typeContract.slug === 'link' &&
+					insertSession.actor.slug !== 'user-admin'
+				) {
```

Comment:
Can drop this? Already have the same check in the `validateOrgMembership()` function.

Reply:
👍